sprout/appliances/tasks.py

```diff
@@ -1584,17 +1589,31 @@ def delete_template_from_provider(self, template_id):
 
 
 @singleton_task()
-def appliance_rename(self, appliance_id, new_name):
-    self.logger.info("Start renaming process for {} to {}".format(appliance_id, new_name))
+def appliance_rename(self, appliance_id):
     try:
         appliance = Appliance.objects.get(id=appliance_id)
+        new_name = '{}_'.format(appliance.appliance_pool.owner.username)
```

Comment:
Consider moving this new code out of the try block since there's no exception handling for anything within these lines, just the appliance object get.